ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -240,138 +240,137 @@
   </Form::Field::ListWrapper>
 
   {{! Account Claim Maps }}
-  <Rose::Form::Fieldset>
-    <:title>{{t 'form.account_claim_maps.label'}}</:title>
-    <:description>{{t 'form.account_claim_maps.help'}}</:description>
-    <:body>
-      {{#if @model.errors.account_claim_maps}}
-        <Rose::Form::Errors as |errors|>
-          {{#each @model.errors.account_claim_maps as |error|}}
-            <errors.message>{{error.message}}</errors.message>
-          {{/each}}
-        </Rose::Form::Errors>
-      {{/if}}
-      <Rose::Table @style='condensed' as |table|>
-        <table.header as |header|>
-          <header.row as |row|>
-            <row.headerCell>{{t 'form.from_claim.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.to_claim.label'}}</row.headerCell>
-            <row.headerCell>{{t 'titles.actions'}}</row.headerCell>
-          </header.row>
-        </table.header>
-        <table.body as |body|>
-          {{#each @model.account_claim_maps as |claimMap i|}}
-            <body.row as |row|>
-              <row.cell>
-                <form.input
+  <Hds::Form::Fieldset @isOptional={{true}} as |F|>
+    <F.Legend>{{t 'form.account_claim_maps.label'}}</F.Legend>
+    <F.HelperText>{{t 'form.account_claim_maps.help'}}</F.HelperText>
+    {{#let (unique-id) (unique-id) as |fromHeaderID toHeaderID|}}
+      <F.Control>
+        <Hds::Table class='list-wrapper-field' name='account_claim_maps'>
```

Comment:
Why we are not using `@columns` to define the table header as we did with all the other HDS tables?

Reply:
@calcaide definitely tried both variations of getting this to work, but I think it is because the table ".data" key doesn't have access to that second table row.  We had to use the same pattern here in [LDAP](https://github.com/hashicorp/boundary-ui/blob/786e6130cd61cdd44d23bca9c554c6ca9dc8d092/ui/admin/app/components/form/auth-method/ldap/index.hbs#L394)

Reply:
@dvonanderson 
Why are we introducing a placeholder? I am not seeing this placeholder on the old table. Also, it what state are you expecting this placeholder to be shown? I tested your PR on 3 states: new, edit and shown, all states.
Also, if you define `@disabled={{@disabled}}` I don't think it will be shown.

Let me know if I am missing something.

Reply:
@calcaide we have used it before in our codebase, and it is currently a part of the `select` component for `Form::Field::ListWrapper`.  I have removed it and I will bring it up for a discussion at tech time.  The general idea is when the user sees the [dropdown](https://github.com/hashicorp/boundary-ui/blob/786e6130cd61cdd44d23bca9c554c6ca9dc8d092/ui/admin/app/components/form/policy/policy-selection/index.hbs#L24) it will show "Choose an Option"  instead of the first item in the options array.  I definitely made a mistake on the `disabled` argument. For now though I have modified the code.

Reply:
I am aware of the place holder functionality and that we have it in our codebase, also examples on HDS documentation. I am not saying we should or shouldn't have it. What I am asking is, in this specific use case and particular component, do we need to have a placeholder?  and since the previous table has no placeholder defined, I am asking what is the reason why we add it here. Did design ask for it? Is an UX improvement? 

I opened Vercel with your implementation:

Edit:
![edit](https://github.com/hashicorp/boundary-ui/assets/9775006/cc9ee3db-50d9-4abe-ba4a-434c9bdc61e2)

List:
![list](https://github.com/hashicorp/boundary-ui/assets/9775006/8b9176e8-de20-458d-a673-7cb5c4c7a621)

New:
![new](https://github.com/hashicorp/boundary-ui/assets/9775006/535157a9-9a02-4ee4-9aeb-d4bcc457da23)

Any of these states show no placeholder, and therefore my question. 

Reply:
@calcaide no UX improvement request from design.  the incorrect value for the `disabled` argument was the reason you didn't see it, but its been removed.